response_operations_ui/views/info.py

```diff
@@ -0,0 +1,23 @@
+import json
+from pathlib import Path
+
+from flask import make_response, jsonify
+
+from response_operations_ui import app
+
+
+@app.route('/info', methods=['GET'])
+def get_info():
+
+    _health_check = {}
+    if Path('git_info').exists():
+        with open('git_info') as io:
+            _health_check = json.loads(io.read())
```

Comment:
probably want to catch the https://docs.python.org/3.6/library/json.html#json.JSONDecodeError here